### PR TITLE
Add health introduction breather page

### PR DESCRIPTION
**Pull request by jessieay**
* [Finishes #152655651]
* This is using less inheritance than other breather pages and can serve
as a model for the refactor described in
https://www.pivotaltracker.com/story/show/152867437 once this is
finalized (to follow)

### CODE REVIEW

File: app/controllers/medicaid/health_introduction_controller.rb [hunk 1]
```diff
@@ -0,0 +1,50 @@
+module Medicaid
+  class HealthIntroductionController < ApplicationController
```
Comment (jessieay) [inline app/controllers/medicaid/health_introduction_controller.rb]: @disambiguator I inherited from `ApplicationController` and only included methods required for a step to work. If we are happy with this, I can update the other breather controllers

Comment (jayroh) [reply app/controllers/medicaid/health_introduction_controller.rb]: If I'm reading this right - would the other breather controllers need everything below this comment?

Comment (jessieay) [reply app/controllers/medicaid/health_introduction_controller.rb]: yep! every controller for a breather page would need all of these methods

Comment (jessieay) [reply app/controllers/medicaid/health_introduction_controller.rb]: where's the :shrug: emoji when you need it!

Comment (jessieay) [reply app/controllers/medicaid/health_introduction_controller.rb]: may be helpful to compare with the controller changes required to add a breather page *with* inheritance https://github.com/codeforamerica/michigan-benefits/pull/481/files#diff-a1c0b16dd0b1db3ce5d021c461d4edba

Comment (jayroh) [reply app/controllers/medicaid/health_introduction_controller.rb]: > yep! every controller for a breather page would need all of these methods

That much duplication, in my opinion, is not ideal. But I'm not entirely sure what the best / right answer is to combat this outside of inheritance or another more composition-based approach. The latter of which I think would need some further discussion with the whole group

**Pull request merged**